src/object/cli_obj.c

```diff
@@ -1843,24 +1910,38 @@ obj_comp_cb(tse_task_t *task, void *data)
 	return 0;
 }
 
+/* register the completion cb for obj IO request */
 static int
-shard_fetch_prep(struct shard_auxi_args *shard_auxi, struct dc_object *obj,
+obj_comp_cb_reg(tse_task_t *task, int opc, uint32_t map_ver,
```

Comment:
code convention is: component + action + postfix, so it should be obj_reg_comp_cb

Reply:
OK, I'll rename it in following patch.